qemu/tests/cfg/rh_kernel_update.cfg

```diff
@@ -2,6 +2,7 @@
     only RHEL
     only Host_RHEL
     only i386 x86_64 PAE ppc64 ppc64le aarch64 s390x
+    only filesystem
```

Comment:
For rhel7, also don't use network storage backend? or just only for rhel8?
if just only for rhel8, I think could add a limited keyword for it.
```
RHEL.8:
    only filesystem
```

Reply:
The change is made for all RHEL, or we also have to change VT code for rh_kernel_update on network storage.